applications/connecting_applications_to_services/binding-workloads-using-sbo.adoc

```diff
@@ -0,0 +1,46 @@
+[id="binding-workloads-using-sbo"]
+= Binding workloads using Service Binding Operator
```

Comment:
Doesn't a `workload` actually mean an `application`?

Reply:
There seems to be a confusion in terms here

Reply:
@pmacik Yes, but when we are describing in general it was decided in our team SBO call meeting that we will use the term workload. And while explaining specific examples and uses cases we will use the term application.